PluginGenerator/RulesPluginBuilder.cs

```diff
@@ -7,187 +7,154 @@
 using SonarQube.Plugins.Common;
 using System;
 using System.Collections.Generic;
+using System.Diagnostics;
```

Comment:
Most the changes are in this file, and a lot of the code has been removed. It's probably simpler just to review the new version rather than comparing it to the previous version.